curator-recipes/src/test/java/org/apache/curator/framework/recipes/cache/TestTreeCache.java

```diff
@@ -377,7 +377,6 @@ public void testKilledSession() throws Exception
         assertEvent(TreeCacheEvent.Type.NODE_ADDED, "/test/me");
 
         KillSession.kill(client.getZookeeperClient().getZooKeeper(), server.getConnectString());
-        assertEvent(TreeCacheEvent.Type.CONNECTION_SUSPENDED);
```

Comment:
Killing the session goes immediately to LOST. It actually was always a possibility but now it does it so fast that it's always the case.